wiki/Article_Style_Guide/en.md

```diff
@@ -276,39 +559,71 @@ Names of images must be somewhat meaningful, please don't use the timestamp or l
 Images that are used for one article are to be placed inside their folders for linking.
 
-It is suggusted to add a subfolder for the images, i.e. `img/` for all images.
-If you have thumbnails and full sized images, it is suggested to use two folders and name the files the same; i.e. `pr/` for preview and `fs/` for full size.
+It is suggusted to add a subfolder for the images, e.g. `img/` for all images.
+If you have thumbnails and full sized images, it is suggested to use two folders and name the files the same; e.g. `pr/` for preview and `fs/` for full size.
 
 Thumbnails are suggested to have a width of 160px and be in the `.jpg` format (unless it has transparency).
 
 All images don't necessarily need to have an alternative text (text that displays if the image fails to load) nor do they need to have hover text.
 
-### Clickable images
+Do **not** give the images any links.
+On a PC platform, they will work most of the time.
+However, on mobile, they will **not** work most of the time as the pointer event triggers the hover text, not the link.
+For example, use `[osu!supporters](/wiki/osu!supporter/)` and not `[![](/wiki/shared/osu!support.png/)](/wiki/osu!supporter/)`.
 
-If when you need an image to be clickable, i.e. click on thumbnail to view a full sized version, you can use this syntax:
+### Flag icons
+
+Flag icons are located inside this folder: `/wiki/shared/flag/`.
+Most of these icons uses the two letter code (first letter is capitalized; second letter is lowercased) and ends with the `.gif` extension.
+A few expections are the multi-flag icons that are in the `.png` format.
+
+For a list of the 2 lettered language codes, see [List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) (the 639-1 column of the table).
+
+When adding a flag inline, use this format:
 
-```md
-[![Alt](thumnbail/link.jpg)](full/link.png "hover")
 ```
+![full-country-name](/wiki/shared/flag/xx.gif)
+```
+
+- `full-country-name` is the full country name
+- `xx.gif` is the two letter code for the flag
 
-- `Alt` is the alternative text; if image fails to load
-- `thumbnail/link.jpg` is the link of the thumbnail
-- `full/link.jpg` is the link of the thumbnail
-- `"hover"` is the hover text; when the mouse cursor is on top of the mouse
+To see if a flag exists, see the `/wiki/shared/flag/` folder.
 
 ## Tables
 
 Tables use the GFM syntax.
 
-If you cannot create a table because _something_ won't work without HTML (i.e. lists inside tables), then you are overthinking both the table and the content.
+If you cannot create a table because _something_ won't work without HTML (e.g. lists inside tables), then you are overthinking both the table and the content.
 If you need help making a complicated table, try to see if you can improvise a simplier fix for it.
 
 **Never** place images inside tables.
 
+It is up to you whether or not if you want to beautify the tables.
```

Comment:
I would recommend it as well, but those whom text wrap will see them looking like this
![image](https://cloud.githubusercontent.com/assets/14264992/24580443/d5d046e6-16cd-11e7-9874-2cb4cf4937b7.png)
instead of
![image](https://cloud.githubusercontent.com/assets/14264992/24580452/ee73f166-16cd-11e7-958a-7a5e3a49bf76.png)